listening-comp/arabic-quiz-app/youtube_extractor.py

```
@@ -0,0 +1,662 @@
+        self.model = SentenceTransformer('all-MiniLM-L6-v2')
+        
+        # Initialize ChromaDB
+        self.db_dir = "chroma_db"
+        os.makedirs(self.db_dir, exist_ok=True)
+        self.client = chromadb.PersistentClient(path=self.db_dir)
+        
+        # Create or get the collection
+        try:
+            self.collection = self.client.get_collection("arabic_quiz_questions")
+            print(f"Loaded existing collection with {self.collection.count()} questions")
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)